### ISSUE TITLE
Can't handle very big SPDX files (e.g.200MB) when read license or make documents

### ISSUE THREAD

**Issue by KoukiHama** [eclipse/sw360#1011]
When user import and get license info from  big size SPDX, sw360 return 500.
![erro500_spdx_check](https://user-images.githubusercontent.com/39894802/95718319-3d287500-0ca9-11eb-9b64-d51a99b71f6f.png)

[bigdatalogsw360.txt](https://github.com/eclipse/sw360/files/5363657/bigdatalogsw360.txt)
commit 749770bd5f78c81d8dd2c0cbf4f4f536082030b0

log: 
>  ERROR ComponentPortlet:569 - Cannot retrieve license information for attachment id 05f05bbf58e9ddb19bd9a7c879003xxx in release xxx92d3768cda7a84950e893cc005350.
> org.apache.thrift.transport.TTransportException: java.net.SocketTimeoutException: Read timed out

and also cant' generate the catalogue ("Generate License Info") with large spdx rdf file
I already report it in [slack](https://sw360chat.slack.com/archives/C68C2RYS3/p1601958324009600)

**Comment by dothanhtrung** [comment]
Hi,

I tried this with a spdx file of linux kernel, and see that program's timeout when it tried to `getAllLicenseTexts` of all `spdxFile` (there are ~70k `spdx:File` in that .rdf).
While website only wants to show `concludedLicenseIds`, can we avoid this and only call `getAllConcludedLicenseIds` instead?

**Comment by dothanhtrung** [comment]
Hi,

I tried latest sw360 (889dab9) with spdx-tools 2.1.20 (and 2.2.3 too). But when I clicked on "Show License Info" button, it failed immediately with error 
> 2020-10-16 07:53:37 ERROR ComponentPortlet:569 - Cannot retrieve license information for attachment id e1988dcdd31af646570de638c100efe5 in release e1988dcdd31a
f646570de638c100d1df.
org.apache.thrift.transport.TTransportException: HTTP Response code: 404

Full error log:
[spdx2.1.20.error.log](https://github.com/eclipse/sw360/files/5390170/spdx2.1.20.error.log)

I changed spdx-tools version in 3 files:
- _backend/src-common/pom.xml_
- _backend/src/src-licenseinfo/pom.xml_
- _backend/src/src-licenses/pom.xml_

**Comment by KoukiHama** [comment]
I wonder this timeout parameter are related to this issue or not.
https://github.com/eclipse/sw360/blob/77730cea56e2229a47efbfddd2656877503b9ae6/libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/thrift/ThriftClients.java#L95-L96

**Comment by KoukiHama** [comment]
I changed above parameter and tried again in 749770b.
5000->50000, 60000->600000 

Error log was changed but could not slove issue.
[changetimeouterror.txt](https://github.com/eclipse/sw360/files/5521558/changetimeouterror.txt)



`2020-11-11 03:17:41 ERROR ComponentPortlet:569 - Cannot retrieve license information for attachment id 9037f0486bd300fbde03981e07057605 in release 9037f0486bd300fbde03981e07054664.
org.apache.thrift.transport.TTransportException: HTTP Response code: 500`
